src/testFixtures/java/org/opensearch/knn/KNNRestTestCase.java

```diff
@@ -761,6 +761,15 @@ protected Settings getKNNSegmentReplicatedIndexSettings() {
             .build();
     }
 
+    protected Settings buildKNNSettingsWithBuildGraphProperty(boolean buildGraph) {
+        return Settings.builder()
+            .put("number_of_shards", 1)
+            .put("number_of_replicas", 1)
```

Comment:
why 1 replica? this will keep the cluster yellow. If you want to test with more than 1 shard do that.